### PR TITLE
Sensu Go Enablement

### PR DESCRIPTION
**Pull request by asachs01**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [x] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [x] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Adds asset capability to plugin collection.

#### Known Compatibility Issues
Drops support for < Ruby 2.3 & requires sensu-plugin 4.0

### CODE REVIEW

**Comment by asachs01** [comment]
ping @cwjohnston & @majormoses

File: README.md [hunk 1]
```diff
@@ -1,24 +1,116 @@
...
 [![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-snmp.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-snmp)
 
-## Functionality
+## Sensu SNMP Plugin
+
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Sensu Go](#sensu-go)
+    - [Asset manifest](#asset-manifest)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
    - [Asset definition](#asset-definition)
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,116 @@
...
+    - [Check manifest](#check-manifest)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
    - [Check definition](#check-definition)
```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,116 @@
...
+#### Sensu Go
+##### Asset registration
+
+Assets are the best way to make use of this handler. If you're not using an asset, please consider doing so! If you're using sensuctl 5.13 or later, you can use the following command to add the asset: 
+
+`sensuctl asset add sensu-plugins/sensu-plugins-snmp`
+
+If you're using an earlier version of sensuctl, you can download the asset definition from [this project's Bonsai Asset Index page](https://bonsai.sensu.io/assets/sensu-plugins/sensu-plugins-snmp).
+
+##### Asset manifest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
##### Asset definition
```

File: README.md [hunk 4]
```diff
@@ -1,24 +1,116 @@
...
+spec:
+  url: https://assets.bonsai.sensu.io/cd39e88ca50bf16793796429a94673df208ddb26/sensu-plugins-snmp_3.0.0-pre_centos_linux_amd64.tar.gz
+  sha512: 7edd57dab82bfd97662d3f2d6ddfb70c41bbcba591b1c2c4077d0107c667fbd2c64ad2bcef51e0192e18176ade60fde403c4784ac141ba1a932cc65c8c897169
+  filters:
+  - entity.system.os == 'linux'
+  - entity.system.arch == 'amd64'
+  - entity.system.platform_family == 'rhel'
+```
+
+##### Check manifest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
##### Check definition
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
I've added a few suggestions to change manifest -> definition. Anchor links may need some help as well, to ensure TOC links go to the right place in the doc.

As this plugin collection depends on the ruby-runtime asset, we also need to call out that dependency.

**Review by majormoses — APPROVED** [review]
LGTM

**Comment by jspaleta** [comment]
Merging, I'll clean up anything small that has drifted

**Pull request merged**